### ISSUE TITLE
Custom colour assignments

### ISSUE THREAD

**Issue by rieder** [rieder/Fresco#18]
When we have support for more wavelengths (e.g. #11, #12), it would be nice to be able to assign these to a specific colour in the resulting image (e.g. X-ray -> blue, visual -> green, IR -> red or combinations thereof, as done in combined observations of Spitzer/Hubble/Chandra).
This is probably too complicated for a command-line interface, so maybe we should be looking to do this in a parameter file or from a GUI.

**Comment by ipelupessy** [comment]
you could, for this you need the skip the mapping from filters to perceptual colors (as its meaningless in that case)..I think one nice thing to do before expanding the wavelength range is to test the resulting images for narrow line sources (Halpha, nebular lines)..it should already work in that case..but is different from broad spectral distributions used until now...

**Comment by rieder** [comment]
Yes, I’d like to add narrowband filters in a future version (probably before implementing this issue). Support for spectral lines (working on that) is probably even more important.
